### PR TITLE
Use low PLL_video when tefix=0 and tefix=2

### PR DESCRIPTION
**Pull request by tiopex**

### CODE REVIEW

**Comment by Apaczer** [comment]
Change Tearing Fix method with following module parameters:
```
tefix=0 //off
tefix=1 //low pll
tefix=2 //old (for higher cpu clocks)
tefix=3 //new (for lower cpu clocks)
```
Applicable through ``miyooctl -t <tefix_method>`` or with gmenu2x settings/editLink

tefix 0 & 1 running at default PLL_VIDEO freq clock (fixes glitch effect on newer devices with this screen)
tefix 2 & 3 running at 2x PLL_VIDEO freq clock speed (for improved Tearing Fix)

File: drivers/video/fbdev/st7789sfb.c [hunk 1]
```diff
@@ -864,22 +915,32 @@ static long myioctl(struct file *filp, unsigned int cmd, unsigned long arg)
...
+            tefix = arg;
+#if defined(DEBUG)
+            printk("st7789sfb: set TE fix to: %d", (int)tefix);
+#endif
+            if (tefix == 3 || tefix == 2) {
+            	writel(0x91001303, iomm.ccm + PLL_VIDEO_CTRL_REG);
+		while((readl(iomm.ccm + PLL_VIDEO_CTRL_REG) & (1 << 28)) == 0){};
+            } else {
+            	writel(0x91001107, iomm.ccm + PLL_VIDEO_CTRL_REG);
+		while((readl(iomm.ccm + PLL_VIDEO_CTRL_REG) & (1 << 28)) == 0){};
```
Comment (tiopex) [inline drivers/video/fbdev/st7789sfb.c:933]: this while is common for both conditions

File: drivers/video/fbdev/st7789sfb.c [hunk 2]
```diff
@@ -864,22 +915,32 @@ static long myioctl(struct file *filp, unsigned int cmd, unsigned long arg)
...
-			}
-			break;				
+        case MIYOO_FB0_SET_TEFIX:
+            tefix = arg;
+#if defined(DEBUG)
+            printk("st7789sfb: set TE fix to: %d", (int)tefix);
+#endif
+            if (tefix == 3 || tefix == 2) {
+            	writel(0x91001303, iomm.ccm + PLL_VIDEO_CTRL_REG);
+		while((readl(iomm.ccm + PLL_VIDEO_CTRL_REG) & (1 << 28)) == 0){};
```
Comment (tiopex) [inline drivers/video/fbdev/st7789sfb.c:930]: This while is common for both cases

**Comment by Apaczer** [comment]
While there were no proper tests on problematic V90, it doesn't explicit issues on older batch of devices with ST7789 screen and leaves everything optional with ``tefix`` parameter.

default ``tefix=0`` should solve performance and compatibility drawbacks.

Merging to not block further development and should be easier to test from beta builds.

**Pull request merged**